cms/djangoapps/contentstore/tasks.py

```diff
@@ -125,3 +126,17 @@ def push_course_update_task(course_key_string, course_subscription_id, course_di
     # TODO Use edx-notifications library instead (MA-638).
     from .push_notification import send_push_course_update
     send_push_course_update(course_key_string, course_subscription_id, course_display_name)
+
+
+@task()
+def async_export_to_git(course_module, user=None):
+    """
+    Exports a course module to Git.
+    """
+    try:
+        export_to_git(course_module.id, course_module.giturl, user=user)
```

Comment:
i don't think we can make that change. `giturl` is the name used by edX for this course setting (https://github.com/mitodl/edx-platform/blob/d25ede5e84334327f675e165c886c1a591ec83c2/common/lib/xmodule/xmodule/modulestore/inheritance.py#L60), and it's referenced in many different parts of the codebase.